modules/persistent-storage-csi-efs-cross-account.adoc

```diff
@@ -0,0 +1,304 @@
++
+* *Type*: NFS
+* *Protocol*: TCP
+* *Port range*: 2049
+* *Source*: Custom/IP address range of your {product-title} cluster nodes (for example, “10.0.0.0/16”)
+
+.. Save the rule.
++
+[NOTE]
+====
+If you encounter mounting issues, re-check the port number, IP address range, and verify that the AWS EFS volume uses the expected security group.
+====
+
+. Create VPC peering between the {product-title} cluster VPC in AWS account A and the AWS EFS VPC in AWS account B:
++
+Ensure the two VPCs are using different network CIDRs, and after creating the VPC peering, add routes in each VPC to connect the two VPC networks.
+
+.. Create a peering connection called, for example, “my-efs-crossaccount-peering-connection” in account B. For the local VPC ID, use the EFS-located VPC. To peer with the VPC for account A, for the VPC ID use the {product-title} cluster VPC ID.
+
+.. Accept the peer connection in AWS account A.
+
+.. Modify the route table of each subnet (EFS-volume used subnets) in AWS account B:
+
+... On the left pane, under *Virtual private cloud*, click the down arrow to expand the available options.
+
+... Under *Virtual private cloud*, click *Route tables"*.
+
+... Click the *Routes* tab.
+
+... Under *Destination*, enter 10.0.0.0/16. 
+
+... Under *Target*, use the peer connection type point from the created peer connection.
+
+.. Modify the route table of each subnet ({product-title} cluster nodes used subnets) in AWS account A:
+
+... On the left pane, under *Virtual private cloud*, click the down arrow to expand the available options.
+
+... Under *Virtual private cloud*, click *Route tables"*.
+
+... Click the *Routes* tab.
+
+... Under *Destination*, enter the CIDR for the VPC in account B, which for this example is 172.20.0.0/16. 
+
+... Under *Target*, use the peer connection type point from the created peer connection.
+
+. Create an IAM role, for example, “my-efs-acrossaccount-role” in AWS account B, which has a trust relationship with AWS account A, and add an inline AWS EFS policy with permissions to call “my-efs-acrossaccount-driver-policy”. 
++
+This role is used by the CSI driver's controller service running on the {product-title} cluster in AWS account A to determine the mount targets for your file system in AWS account B.
++
+[source, json]
+----
+# Trust relationships trusted entity trusted account A configuration on my-efs-acrossaccount-role in account B
+
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::301721915996:root"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {}
+        }
+    ]
+}
+
+# Trust relationships trusted entity trusted account A configuration on my-efs-acrossaccount-role in account B
```

Comment:
@lpettyjo Can you plz remove Line L131 to L145. Bec L115 to L129 are same with L131 to L145/added two times.
Thanks. Others LGTM.

Reply:
Ha. Good catch indeed. Fixed.